npm install doesn't work on node 7.x.x

I have `node 7.x.x`, it gives the following error. To fix, I reverted it to `6.9.1`. Works great now! But please fix this for `node 7.x.x`:

```bash
> snowboy@1.1.0 install /Users/dolan/DefinitelyTyped/node_modules/snowboy
> node-pre-gyp install --fallback-to-build

node-pre-gyp ERR! Tried to download(403): https://snowboy-release-node.s3-us-west-2.amazonaws.com/snowboy/v1.1.0/Release/snowboy-v1.1.0-node-v51-darwin-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for snowboy@1.1.0 and node@7.0.0 (node-v51 ABI) (falling back to source compile with node-gyp) 
module.js:474
    throw err;
    ^

Error: Cannot find module 'nan'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at [eval]:1:1
    at ContextifyScript.Script.runInThisContext (vm.js:25:33)
    at Object.exports.runInThisContext (vm.js:77:17)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:573:32)
    at bootstrap_node.js:345:29
gyp: Call to 'node -e "require('nan')"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -35,6 +35,7 @@ env:
     - NODE_VERSION="4.0.0"
     - NODE_VERSION="5.0.0"
     - NODE_VERSION="6.0.0"
+    - NODE_VERSION="7.0.0"
 
 before_install:
 # use the correct version of node

